bin/k8s_condor_cpu_efficiency.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Create a site for the default period for each of the cms types
+#
+# This script is intended to be used as cron job in kubernetes.
+
+# Kerberos
+keytab=/etc/condor-cpu-eff/keytab
+principal=`klist -k "$keytab" | tail -1 | awk '{print $2}'`
+echo "principal=$principal"
+kinit $principal -k -t "$keytab"
+if [ $? == 1 ]; then
+    echo "Unable to perform kinit"
+    exit 1
+fi
+
+currentDir=$(
+  cd "$(dirname "$0")" && pwd
+)
+if ! klist -s
+then
+    echo "There is no valid kerberos ticket yet"
+    kinit
+fi
+
+# check output parameter
+[[ -z "$1" ]] && { echo "ERROR: Output path is not defined" ; exit 1; }
+
+OUTPUT_DIR="$1"
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+CMS_TYPES=("analysis" "production" "folding@home" "test")
+
+spark_submit=/usr/hdp/spark-2.3.0/bin/spark-submit
+spark_confs=(
+    --conf spark.driver.bindAddress=0.0.0.0
+    --conf spark.driver.host=$MY_NODE_NAME
```

Comment:
Usage of custom env variable should be checked first. Please add block to check that MY_NODE_NAME and CONDOR_CPU_EFF_SERVICE_PORT_PORT_0 are defined. If not the script should exit with error status.